interop-tests/src/test/scala/io/grpc/testing/integration/AkkaGrpcJavaClientTester.java

```diff
@@ -17,6 +17,7 @@
 import com.google.protobuf.ByteString;
 import io.grpc.Status;
 import io.grpc.StatusRuntimeException;
+import io.grpc.testing.integration.test.SSLContextUtils;
```

Comment:
For now we have one utility method in our tests. In the future we might make a helper class somewhere for users to use.